Week3/homework/step3.js

```diff
@@ -3,6 +3,10 @@
 function createBase(base) {
   // Replace this comment and the next line with your code
   console.log(base);
+  return function addSix(number) {
```

Comment:
Good. You don't need to name this function `addSix` though - you could leave it as an anonymous function.